WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
Regarding whether or not we'd be ok to deprecate the "old" whitelist tag in the future:

I just ran a search for `tax_query ok` on a mirror of both the theme repository as well as the plugin repository.

Results:
* themes: no usage found in 4.000+ themes
* plugins: used in 1 file in 1 plugin out of 55.000+ plugins

Of course, this is excluding any themes and plugins not hosted on wordpress.org, but I think it gives a clear indication of how little this whitelist comment is currently used :sunglasses: